README.md

```diff
@@ -105,6 +105,14 @@ location /whiteboard/ {
 }
 ```
 
+#### Caddy v2
+
+```
+handle_path /whiteboard/* {
+    reverse_proxy http://127.0.0.1:3002
```

Comment:
AIO seems to also strip the path from the URI passed along. Should both work as of right now, but might be safer that way:

https://github.com/nextcloud/all-in-one/blob/9130cff8041bcb96b49ec67a50cbc6403f22e235/Containers/apache/Caddyfile#L55-L58

Reply:
It is semantically the same. But we can have it more explicit if you prefer.

> Works the same as the [handle directive](https://caddyserver.com/docs/caddyfile/directives/handle), but implicitly uses [uri strip_prefix](https://caddyserver.com/docs/caddyfile/directives/uri) to strip the matched path prefix.

Ref https://caddyserver.com/docs/caddyfile/directives/handle_path#examples